app/i18n/en/conf.php

```diff
@@ -177,6 +177,12 @@
 				'header' => 'In header',
 				'none' => 'None',
 			),
+			'icons' => array(
+				'_' => 'Icons',
```

Comment:
IMHO: `Icons` is to general. Something like "mark (un)read, favorite buttons" would describe it more precisely

Reply:
I agree - I wasn't sure what to put for this. Maybe instead even `Article actions`. This would allow these to be replaced with text in some future change / (or theme).

Reply:
This might be a better UI for the configuration of these things in the reader view too.